src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -6308,7 +6309,7 @@ html[dir][data-theme=default] .toast-body {
 html[data-theme=default] .modal {
   position: fixed;
   top: 0;
-  z-index: 1055;
+  z-index: 1200;
```

Comment:
1055 is greater than `$zindex-modal`?

Reply:
navbar is ($zindex-modal + 100) and the leftmenu is ($zindex-modal + 50). The modal need to be greater than ($zindex-modal + 100) so in this case `1150` is good to use. Check out the updated comments

Reply:
I might need to test it, but I never mind If someone approves after a proper test